packages/portal/src/mixins/klaro.js

```diff
@@ -66,12 +70,30 @@ export default {
   },
 
   methods: {
+    onKlaroScriptLoad() {
+      if (!this.klaro) {
+        this.klaro = window.klaro;
+      }
+
+      // If Matomo plugin is installed, wait for Matomo to load, but still render
+      // Klaro if it fails to.
+      const renderKlaroAfter = this.$waitForMatomo ? this.$waitForMatomo() : Promise.resolve();
+
+      renderKlaroAfter.catch(() => {}).finally(this.renderKlaro);
+    },
+
+    onKlaroRendered() {
+      // may be overriden by components including the mixin
+    },
```

Comment:
This does nothing. Is it just for reference?

Reply:
No, but refactored so that it's redundant.